src/gc/env/gcenv.os.h

```diff
@@ -277,6 +277,12 @@ class GCToOSInterface
     // Return:
     //  Time stamp in milliseconds
     static uint32_t GetLowPrecisionTimeStamp();
+
+    // Gets the current instruction pointer.
+    // Return:
+    //  The value of the instruction pointer immediately after
+    //  the call to this function.
+    static void* GetCurrentInstructionPointer();
```

Comment:
Ok, will do! I noticed that we're only calling it twice and with two different error codes too, so that sounds reasonable to me.